{"time":"2025-06-24T12:09:04.4930598-04:00","level":"INFO","msg":"stream: starting","core version":"0.20.1","symlink path":"c:\\Users\\LiamMurphy(DMC)\\Documents\\Weave\\eval\\wandb\\run-20250624_120903-jfq5ythl\\logs\\debug-core.log"}
{"time":"2025-06-24T12:09:06.7799291-04:00","level":"INFO","msg":"stream: created new stream","id":"jfq5ythl"}
{"time":"2025-06-24T12:09:06.7805874-04:00","level":"INFO","msg":"stream: started","id":"jfq5ythl"}
{"time":"2025-06-24T12:09:06.7805874-04:00","level":"INFO","msg":"handler: started","stream_id":"jfq5ythl"}
{"time":"2025-06-24T12:09:06.7805874-04:00","level":"INFO","msg":"sender: started","stream_id":"jfq5ythl"}
{"time":"2025-06-24T12:09:06.7805874-04:00","level":"INFO","msg":"writer: Do: started","stream_id":"jfq5ythl"}
{"time":"2025-06-24T12:09:06.9887292-04:00","level":"INFO","msg":"Starting system monitor"}
{"time":"2025-06-24T12:12:02.2270451-04:00","level":"INFO","msg":"api: retrying HTTP error","status":500,"url":"https://weave.assessmentforgood.cloud/graphql","body":"{\"errors\":[{\"message\":\"An internal error occurred. Please contact support.\",\"path\":[\"project\",\"run\"]}],\"data\":{\"project\":{\"run\":null}}}"}
{"time":"2025-06-24T12:12:05.2440994-04:00","level":"INFO","msg":"api: retrying HTTP error","status":500,"url":"https://weave.assessmentforgood.cloud/files/liammurphy7657/dataset-severitylevels-104/jfq5ythl/file_stream","body":"{\"error\":\"dial tcp 10.101.166.169:3306: connect: connection refused\"}"}
{"time":"2025-06-24T12:12:06.6825667-04:00","level":"INFO","msg":"api: retrying HTTP error","status":500,"url":"https://weave.assessmentforgood.cloud/graphql","body":"{\"errors\":[{\"message\":\"dial tcp 10.101.166.169:3306: connect: connection refused\",\"path\":[\"project\"]}],\"data\":{\"project\":null}}"}
{"time":"2025-06-24T12:12:07.8993039-04:00","level":"INFO","msg":"api: retrying HTTP error","status":500,"url":"https://weave.assessmentforgood.cloud/files/liammurphy7657/dataset-severitylevels-104/jfq5ythl/file_stream","body":"{\"error\":\"dial tcp 10.101.166.169:3306: connect: connection refused\"}"}
{"time":"2025-06-24T12:12:12.0120878-04:00","level":"INFO","msg":"api: retrying HTTP error","status":500,"url":"https://weave.assessmentforgood.cloud/graphql","body":"{\"errors\":[{\"message\":\"dial tcp 10.101.166.169:3306: connect: connection refused\",\"path\":[\"project\"]}],\"data\":{\"project\":null}}"}
{"time":"2025-06-24T12:12:22.8482705-04:00","level":"INFO","msg":"api: retrying HTTP error","status":500,"url":"https://weave.assessmentforgood.cloud/files/liammurphy7657/dataset-severitylevels-104/jfq5ythl/file_stream","body":"{\"error\":\"dial tcp 10.101.166.169:3306: i/o timeout\"}"}
{"time":"2025-06-24T13:07:21.2749327-04:00","level":"INFO","msg":"api: retrying error","error":"Post \"https://weave.assessmentforgood.cloud/files/liammurphy7657/dataset-severitylevels-104/jfq5ythl/file_stream\": dial tcp: lookup weave.assessmentforgood.cloud: no such host"}
{"time":"2025-06-24T13:07:23.6326246-04:00","level":"INFO","msg":"api: retrying error","error":"Post \"https://weave.assessmentforgood.cloud/files/liammurphy7657/dataset-severitylevels-104/jfq5ythl/file_stream\": dial tcp: lookup weave.assessmentforgood.cloud: no such host"}
{"time":"2025-06-24T13:14:49.3595107-04:00","level":"INFO","msg":"stream: closing","id":"jfq5ythl"}
{"time":"2025-06-24T13:14:49.3595107-04:00","level":"INFO","msg":"Stopping system monitor"}
{"time":"2025-06-24T13:14:49.3600291-04:00","level":"INFO","msg":"Stopped system monitor"}
{"time":"2025-06-24T13:14:50.7925886-04:00","level":"INFO","msg":"fileTransfer: Close: file transfer manager closed"}
{"time":"2025-06-24T13:14:50.8955875-04:00","level":"INFO","msg":"handler: closed","stream_id":"jfq5ythl"}
{"time":"2025-06-24T13:14:50.8955875-04:00","level":"INFO","msg":"writer: Close: closed","stream_id":"jfq5ythl"}
{"time":"2025-06-24T13:14:50.8955875-04:00","level":"INFO","msg":"sender: closed","stream_id":"jfq5ythl"}
{"time":"2025-06-24T13:14:50.8955875-04:00","level":"INFO","msg":"stream: closed","id":"jfq5ythl"}
